mod/gc_communities/views/default/page/layouts/index.php

```diff
@@ -34,24 +34,46 @@
 
 <div class="row">
     <div class="col-md-8">
-        <?php
-
+        <?php      
         $options = array(
-                'type' => 'object',
-                'subtypes' => $subtypes,
-                'limit' => $newsfeed_limit,
-                'metadata_name' => 'audience',
-                'metadata_values' => $community_audience, //We can add allps metadata here
-                'full_view' => false,
-                'list_type_toggle' => false,
-                'pagination' => true
-            );
-
+            'type' => 'object',
+            'subtypes' => $subtypes,
+            'limit' => $newsfeed_limit,
+            'full_view' => false,
+            'list_type_toggle' => false,
+            'pagination' => true
+            
+        );
+        $tags_values ='';
+        $tags_display ='';
+        if( strpos($community_tags, ',') !== false ){ // if multiple tags
+                $community_tags = array_map('trim', explode(',', $community_tags));
+            foreach($community_tags as $tag_val){
+                $tags_value = elgg_get_metastring_id($tag_val);
+                $tags_values .= ' OR md.value_id = '.$tags_value;

```

Comment:
can use IN (implode(',', $community_tags)) instead of looping over them all

Reply:
I feel I still may need a loop to get the metastrings (the line above).